src/runtime/route.ts

```diff
@@ -133,21 +133,36 @@ export function functionToRoute(
     debug('Event for %s: %o', req.path, event);
     const context = constructContext(config);
     debug('Context for %s: %p', req.path, context);
+    let run_timings: {
+      start: [number, number];
+      end: [number, number];
+    } = {
+      start: [0, 0],
+      end: [0, 0],
+    };
 
     const callback: ServerlessCallback = function callback(
       err,
       responseObject?
     ) {
+      run_timings.end = process.hrtime();
       debug('Function execution %s finished', req.path);
       if (err) {
         handleError(err, req, res, functionFilePath);
         return;
       }
       handleSuccess(responseObject, res);
+      debug(
```

Comment:
Should we have this on line 150 before the error handler? I feel like this is useful even if an error happens as the error might be intentional

Reply:
Yes, I did overlook the return statement on 152, oops.